handler/scheduler.go

```diff
@@ -156,3 +149,16 @@ func scheduleTask(s *eremeticScheduler, request types.Request) (string, error) {
 	s.tasks <- task.ID
 	return task.ID, nil
 }
+
+func updateStatusForTask(status *mesos.TaskStatus) {
+	id := status.TaskId.GetValue()
+	log.Debugf("TaskId [%s] status [%s]", id, status.State)
+	task, _ := database.ReadTask(id)
+	newStatus := types.Status{
+		Status: status.State.String(),
+		Time:   time.Now().Unix(),
+	}
+	task.Status = append([]types.Status{newStatus}, task.Status...)
```

Comment:
hmm ok. I guess it comes down to what the "right order" is :cat:

I do like the reverse chronological order in the presentation